src/PowerBIEmbedded/PowerBI.Test/ScenarioTests/PowerBITests.ps1

```diff
@@ -236,3 +236,76 @@ function Test-NegativePowerBIEmbeddedCapacity
 		Invoke-HandledCmdlet -Command {Remove-AzResourceGroup -Name $resourceGroupName -ErrorAction SilentlyContinue} -IgnoreFailures
 	}
 }
+
+<#
+.SYNOPSIS
+Tests PowerBI Embedded Capacity lifecycle for large skus (A7,A8) (Create, Update, Scale, Get, Delete).
+#>
+function Test-PowerBIEmbeddedCapacityLargeSku
+{
+	try
+	{  
+		# Creating capacity
+		$RGlocation = Get-RG-Location
+		$location = Get-Location
+		$resourceGroupName = Get-ResourceGroupName
+		$capacityName = Get-PowerBIEmbeddedCapacityName
+
+		New-AzResourceGroup -Name $resourceGroupName -Location $RGlocation
+		
+		$capacityCreated = New-AzPowerBIEmbeddedCapacity -ResourceGroupName $resourceGroupName -Name $capacityName -Location $location -Sku 'A7' -Administrator 'aztest0@stabletest.ccsctp.net','aztest1@stabletest.ccsctp.net'
+    
+		Assert-AreEqual $capacityName $capacityCreated.Name
+		Assert-AreEqual $location $capacityCreated.Location
+		Assert-AreEqual "Microsoft.PowerBIDedicated/capacities" $capacityCreated.Type
+		Assert-AreEqual 2 $capacityCreated.Administrator.Count
+		Assert-True {$capacityCreated.Id -like "*$resourceGroupName*"}
+	
+		[array]$capacityGet = Get-AzPowerBIEmbeddedCapacity -ResourceGroupName $resourceGroupName -Name $capacityName
+		$capacityGetItem = $capacityGet[0]
+		Assert-True {$capacityGetItem.State -like "Succeeded"}
+
+		# Test to make sure the capacity does exist
+		Assert-True {Test-AzPowerBIEmbeddedCapacity -Name $capacityName}
+		# Test it without specifying a resource group
+		Assert-True {Test-AzPowerBIEmbeddedCapacity -Name $capacityName}
+		
+		# Updating capacity and Scale up A7 -> A5
+		# scaling to A7/A8 is not supported in this region since the list of skus returns A1-A6.
```

Comment:
What region is this test creating capacities in?
So new PowrShell cmdlet code allows A7 / A8 regardless of the region but the reqest may fail based on the availability of A7 / A8?